src/app/components/AddTodo.tsx

```
@@ -7,31 +7,32 @@
   const [text, setText] = useState("");
   const [isLoading, setIsLoading] = useState(false);
   const [error, setError] = useState<string | null>(null);
+  const [unusedState, setUnusedState] = useState(false); // Unused state variable
```

ESLint

[error] 'unusedState' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)